<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WTaxAttack</class>
 <widget class="QWidget" name="WTaxAttack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="8">
       <widget class="QPushButton" name="btnPrintTax">
        <property name="text">
         <string>Start!</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/printer.png</normaloff>:/images/printer.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Cardex</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="EQLineEdit" name="leCardexName">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="EQLineEdit" name="leCardex">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="EDateEdit" name="deStart">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>283</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="btnRefresh">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="EDateEdit" name="deEnd">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Checkout date</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Common search</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="EQLineEdit" name="leSearch"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Invoices</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="2">
       <widget class="EQLineEdit" name="leInvoice">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="EQTableWidget" name="tbl">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>Invoice</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Checkout </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Guest</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cardex</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Voucher</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Purpose</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ItemCode</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mode of VAT</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="EQTableWidget" name="tblt">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>New Column</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">eqtablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
